spec/models/plant_accession_spec.rb

```diff
@@ -71,5 +90,17 @@
       expect(pa1).to be_valid
       expect(pa2).to be_valid
     end
+
+    it 'properly serializes pv.name' do
```

Comment:
My spec essentially duplicates your spec (guess we both committed at the same time). I think one of those could safely be removed.